### ISSUE TITLE
jqPlot anyone?

### ISSUE THREAD

**Issue by smartinsightsfromdata** [timelyportfolio/buildingwidgets#60]
I saw this interesting library here: http://www.jqplot.com/index.php

But I didn't look at it in any details.  

@timelyportfolio  Maybe it would be worth your expert look to determine the complexity of an integration?

(incidentally, I think It would be useful if there was a list of projects people are working to, to determine priorities etc.)

**Comment by timelyportfolio** [comment]
I agree it is a good little library with some nice touches.  For me, I think I am going to lay off the comprehensive chart libraries for the rest of the year and probably focus on the more specialized types.  Happy to assist if somebody is ambitious.

Are there any particular types or features from `jqPlot` that you are interested in?

**Comment by smartinsightsfromdata** [comment]
I found it researching funnel charts but sadly there is no example of their funnels on their site :)

I fully agree with you re: ambitious projects: I'm in the same boat.
